cli/cmd/logout.go

```diff
@@ -41,7 +44,7 @@ func newCmdLogout() *cobra.Command {
 }
 
 func runLogout(_ context.Context, cmd *cobra.Command) error {
-	err := auth.Logout()
+	err := errors.Join(auth.Logout(), authpb.RemoveRefreshToken(), config.UnsetValue("team"))
```

Comment:
Looks like `auth.Logout()` is doing `RemoveRefreshToken()` and `config.UnsetValue("team")`, but sequentially and stopping on errors rather than sequentially and not stopping on errors.

Shouldn't we just change the behaviour of logout? 🤔

Reply:
335ee72579983358e88f08263a4d8355739cec9f